core/control-plane/control-plane-aggregate-services/src/main/java/org/eclipse/edc/connector/service/query/QueryValidator.java

```diff
@@ -40,9 +42,9 @@ public class QueryValidator {
     /**
      * Constructs a new QueryValidator instance.
      *
-     * @param canonicalType The Java class of the object to validate against.
+     * @param canonicalType    The Java class of the object to validate against.
```

Comment:
Formatting incorrect, please fix this and the line below

Reply:
done